rocksdb_admin/admin_handler.cpp

```diff
@@ -786,6 +786,8 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
   clearMetaData(request->db_name);
   if (!FLAGS_rocksdb_allow_overlapping_keys) {
     // clear DB if overlapping keys are not allowed
+    auto db_role = db->IsSlave() ?
```

Comment:
when we allow overlapping keys, the seqno will increment, but we won't be able to call GetUpdatesSince(), whats the reason for this change?

Reply:
sorry I actually missed the ! in the condition